core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -750,6 +750,14 @@ class TaskSetManagerSuite
     assert(thrown2.getMessage().contains("bigger than spark.driver.maxResultSize"))
   }
 
+  test("do not check total size of intermediate stages") {
```

Comment:
Shall we add a JIRA prefix in the title? `SPARK-32470: do not ...` It's actually in the guidelines.